packages/api/src/models/medical/patient.ts

```diff
@@ -18,6 +19,9 @@ export class PatientModel extends BaseModel<PatientModel> implements Patient {
         facilityIds: {
           type: DataTypes.ARRAY(DataTypes.STRING),
         },
+        externalId: {
+          type: DataTypes.STRING,
+        },
```

Comment:
Missing optional here

Reply:
Arent they optional by default?

Reply:
Yeah, you're right, it's optional by default - I thought it was required by default.

With this just realized that some tables have optional columns that should be required. This is the reason I recommend folks to be explicit about table declaration.

I'll ticket it for other tables, but I suggest we start being explicit about column requirement.